packages/ckeditor5-utils/src/env.ts

```diff
@@ -9,11 +9,13 @@
  * @module utils/env
  */
 
+import globalVar from './dom/global.js';
```

Comment:
```suggestion
import global from './dom/global.js';
```

I think we always use this convention.

Reply:
Fixed.